App/src/pages/addUserToGroup.tsx

```diff
@@ -0,0 +1,157 @@
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    }
+    useEffect(() => {
+        if (location.state) {
+            const _state = location.state as { gid: string; name: string };
+            setLocationState(_state);
+            getMyGroup(location.state.gid);
+            getUsers();
+        }
+    }, [location]);
+
+    useEffect(() => {
+        const filteredUsers = users.filter((user) => {
+            if (user.roles.find((role) => role.userRole === UserRoleCodes.Admin)) return false;
+            if (myGroup.members.find((member) => member.id === user.id)) return false;
+
+            return true;
+        });
+        setNonMembers(filteredUsers);
+    }, [users, myGroup]);
+
+    function addingUser(uid: string) {
+        AddUserToGroup(myGroup.id, uid)
+            .then(
+                () => {
+                    getMyGroup(myGroup.id);
+                },
+                (rej) => {
+                    console.log(rej);
+                }
+            )
+            .catch((err) => {
+                console.log(err);
+            });
+    }
+
+    return (
+        <Container maxW="lg" py={{ base: "12", md: "24" }} px={{ base: "0", sm: "8" }}>
+            <Stack spacing="8">
+                <Box
+                    py={{ base: "0", sm: "8" }}
+                    px={{ base: "4", sm: "10" }}
+                    bg={{ base: "transparent", sm: "bg.surface" }}
+                    boxShadow={{ base: "none", sm: "2xl" }}
+                    borderRadius={{ base: "none", sm: "xl" }}
+                >
+                    <Stack spacing="6">
+                        <img src={logo} alt="Logo" width="50px" height="50px" style={{ margin: "auto" }} />
+                        <Heading textAlign="center" size="lg">
+                            Add User to Group: {locationState.name}
+                        </Heading>
+
+                        <FormControl>
+                            <FormLabel htmlFor="search-text">Search:</FormLabel>
+                            <Input id="searcher" type="text" />
+                        </FormControl>
+
+                        <div id="item-list">
+                            <ol>
+                                {nonMembers.map((item, index) => (
+                                    <HStack justify="space-between" key={index}>
+                                        <li>{item.email}</li>
+                                        <Button
+                                            style="primary"
+                                            variant="outline"
+                                            width="22%"
+                                            height="40px"
+                                            justifyContent={"flex-end"}
+                                            onClick={() => addingUser(item.id)}
+                                        >
+                                            Select
```

Comment:
Can you change the button content to "Add"?

Reply:
sure